ui/src/components/utils.tsx

```diff
@@ -5,7 +5,7 @@ import { DEFAULT_COMPONENT_TYPE_ID } from '../constants';
 import { COMPONENT_TYPES } from './assets';
 
 export const capitalize = (value: string): string =>
```

Comment:
I dont love that it is called capitalize, but then does more than just capitalize 😅

Reply:
Agreed and fixed by renaming to `capitalizeAndReplaceUnderscoresWithSpaces`